templates/service/conf/application.conf

```diff
@@ -28,11 +28,6 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 # Custom error handler
 play.http.errorHandler = "uk.gov.hmrc.$!APP_PACKAGE_NAME!$.config.ErrorHandler"
 
-# Default strict CSP from https://www.playframework.com/documentation/2.8.x/CspFilter#Default-CSP-Policy
-# with an SHA hash to allow the Javascript-detection inline script from govuk-frontend:
-# https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#if-your-javascript-is-not-working-properly
-play.filters.csp.directives.script-src = ${play.filters.csp.nonce.pattern} "'self' 'unsafe-inline' 'strict-dynamic' 'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU=' https: http:"
-
```

Comment:
@JoPintoPaul was it the whole bit of config we wanted to remove or just the `'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU='` bit? If we remove `${play.filters.csp.nonce.pattern} `then I think the nonce attribute we add to scripts will stop working. I may well have miss understood something about this change!

Reply:
@oscarduignan I had it in my head that this configuration was now being done centrally for services BUT I am going to double-check this!

Reply:
@oscarduignan Confirmed via documentation and in code, the `${play.filters.csp.nonce.pattern}` and the rest of the default directives is pulled in as per https://www.playframework.com/documentation/2.8.x/CspFilter#Default-CSP-Policy.